src/ThemeProvider.tsx

```diff
@@ -167,10 +172,10 @@ function applyColorScheme(theme: Theme, colorScheme: string) {
 
     // Apply the first defined color scheme
     const defaultColorScheme = Object.keys(theme.colorSchemes)[0]
-    return deepmerge(theme, theme.colorSchemes[defaultColorScheme])
+    return [deepmerge(theme, theme.colorSchemes[defaultColorScheme]), defaultColorScheme] as const
```

Comment:
Probably out of scope for this PR, but should we have smarter defaults for color schemes?

For example, we could default `light_foo` to `light` but `dark_foo` to `dark`?

Reply:
agreed, I think that would be nice for a future tweak here!